examples/flow/flow_pois.py

```diff
@@ -58,6 +61,33 @@ def get_set(file_name):
 wrapper = LammpsWrapper()
 wrapper.CM[CUBA.NUMBEROF_TIME_STEPS] = 10000
 wrapper.CM[CUBA.TIME_STEP] = 0.003
+wrapper.CM[CUBA.PAIR_POTENTIALS] = ("lj:\n"
```

Comment:
To my understanding of the MD world, a user can configure how two types of atoms interact using these pair potentials.  In the test example there are 3-types so the user configures how he wants these different types to interact with each other.   

I don't think there is a finite set of configurations that a user can select from;  so the user needs to be able to configure the pairs potentials at runtime.    @ahashibon, can you correct me if i what i am saying is wrong